director_infos.py

```diff
@@ -0,0 +1,5 @@
+class Director:
```

Comment:
Create a class People and use it for both actors and directors

Reply:
Done! Thanks!